http://bugs.gentoo.org/219729

commit 697e28939b7d0a3e0ffe3b6bd516213a55f5a063
Author: Jeff Layton <jlaton@redhat.com>
Date:   Mon Apr 14 09:03:13 2008 -0400

    Change how mount.nfs handles EACCES errors. Currently,
    EACCES is a non-fatal error which means the mount will be
    retied. This caused mounts to hang for 2mins when the client
    does not have permission to access the export. In a strict
    interpretation, the error that should be returned is EPERM, but
    this is not always the case. So due to the fuzzy interpretation,
    of EPERM and EACCES, EACCESS is now a fatal error
    
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/utils/mount/stropts.c b/utils/mount/stropts.c
index cadb1f4..cdd610e 100644
--- a/utils/mount/stropts.c
+++ b/utils/mount/stropts.c
@@ -252,7 +252,6 @@ static int set_mandatory_options(const char *type,
 static int is_permanent_error(int error)
 {
 	switch (error) {
-	case EACCES:
 	case ESTALE:
 	case ETIMEDOUT:
 	case ECONNREFUSED:
